homework-2/scss/_sliderlogo.scss

```diff
@@ -0,0 +1,82 @@
+@import 'vars';
+@import 'mixins';
+
+.slider-logo {
+  padding: 100px 0;
+  .container{
+    @include widescreen {
+      padding: 0 200px;
+    }
+  }
+  @include widescreen {
+    padding: 145px 0;
+  }
+}
+.slider-logo-single {
+  .slick-slide .single-img {
+    max-width: 235px;
+    width: 100%;
+    margin: 0 auto;
+  }
+  .slick-disabled {
+    opacity: 0.5;
+  }
+}
+.slider-logo-list {
+  margin-top: 40px;
+  .img-logo { 
+    max-width: 70px;   
+    width: 100%;
+    margin: auto;
+    padding-bottom: 15px;
+    @include desktop {
+      max-width: 120px;
+    }
+  }
+}
+.slider-logo-list .slick-slide {
+  border-bottom: 1px solid #e5e5e5;
+}
+.slider-logo-list .slick-slide.slick-current.slick-active {
+  border-bottom: 5px solid $brandColor;
+  border-radius: 2px;
+}
+.slider-logo-description {
+  max-width: 315px;
+  width: 100%;
+  margin: 0 auto;
+  @include tablet {
+    max-width: 700px;
+  }
+  @include desktop {
+    max-width: 1033px;
+  }
+  @include widescreen {
+    max-width: 570px;
```

Comment:
hardcoded width again. + it reduces when screen goes wider - this is strange.

Reply:
change